content/applications/hr/payroll/contracts.rst

```diff
@@ -0,0 +1,250 @@
+  specifically written for them.
+- :guilabel:`Who can Sign`: Select either :guilabel:`All Users` or :guilabel:`On Invitation`.
+
+  - :guilabel:`All Users`: Any user in the organization can sign the contract.
+  - :guilabel:`On Invitation`: Only users selected in this field can sign the contract.
+
+- :guilabel:`Invited Users`: Select the person(s) that can sign the document.
+- :guilabel:`Document`: The attached document can be replaced by clicking the
+  :guilabel:`✏️ (Select)` icon. A pop-up window appears so another document can be selected for
+  upload. The file must be a PDF. To remove the document, click the :guilabel:`🗑️ (Clear)` icon.
+
+Once the edits are complete, click the :guilabel:`Save` button. All the information for the selected
+contract template populates the fields in the :guilabel:`Salary Information` tab. Any additional
+tabs, such as :guilabel:`Personal Documents`, appears if applicable.
+
+Salary information
+------------------
+
+.. image:: contracts/salary-info.png
+   :align: center
+   :alt: Optional tabs for a new contract.
+
+This section is where the specific salary details are defined. This section is country-specific, so
+depending on where the company is located, these fields may vary.
+
+Enter the :guilabel:`amount` in the various fields, or :guilabel:`check a box` to apply a benefit.
+Some options that can be entered here include :guilabel:`Meal Vouchers`, :guilabel:`Fuel Card`,
+:guilabel:`Internet`, :guilabel:`Paid Time Off`, etc.
+
+Some fields may be automatically filled in based off of the contracts selected in the
+:guilabel:`Contract Details` tab.
+
+Attachment of salary
+--------------------
+
+.. image:: contracts/salary-attachment.png
+   :align: center
+   :alt: Optional tabs for a new contract.
+
+Any automatic deductions or allocations for an employee, such as child support payments and wage
+garnishments, are referred to as a *salary attachment*. This section is where all of these
+deductions or allocations are set.
+
+To add a new deduction, click :guilabel:`Add a line`. Type in a description for the allocation under
+:guilabel:`Description`.
+
+.. image:: contracts/garnishment.png
+   :align: center
+   :alt: Enter a new line for each type of garnishment.
+
+Select the :guilabel:`Garnished Type` from the drop-down menu. Choose from:
+
+- :guilabel:`Attachment of Salary`: Any payments taken out towards something that is *not* child
+  support. Typically any garnishments such as lawsuit payments, payments toward taxes owed, etc.
+- :guilabel:`Assignment of Salary`: Any deduction that is not required but voluntary, such as a
+  pre-tax allocation to a college savings account.
+- :guilabel:`Child Support`: Any payments taken out specifically for child support.
+
+Enter the start and end dates the entry applies to. Click on the :guilabel:`drop-down menu` under
+:guilabel:`From` and :guilabel:`To`, navigate to the correct month and year by using the arrow
+icons, then click on the :guilabel:`date`.
```

Comment:
I mixed up the elements that needed :guilabel:s here in my previous suggestion, please use this corrected RST:
```suggestion
Enter the start and end dates the entry applies to. Click on the drop-down menu under
:guilabel:`From` and :guilabel:`To`, navigate to the correct month and year by using the
:guilabel:`< > (arrow)` icons, then click on the :guilabel:`date`.
```